pkgs/development/python-modules/wxPython/4.0.nix

```
@@ -1,16 +1,31 @@
 { lib
+, stdenv
+, openglSupport ? true
+, libX11
+, pyopengl
 , buildPythonPackage
 , fetchPypi
 , pkgconfig
-, gtk3
 , libjpeg
 , libtiff
+, wxmac
+, darwin
 , SDL
 , gst-plugins-base
 , libnotify
 , freeglut
 , xorg
 , which
+, cairo
+, pango
+, pathlib2
+, python3
```

inherit python from parent scope, this way you can build python2 and python3
```suggestion
, python
```